README.md

```diff
@@ -317,3 +317,44 @@ Delete the service resources by running the following ensuring your reference th
 
   terraform destroy -var-file=env/dev/dev.tfvars
   ```
+
+  ## Linting and Formatting
```

Comment:
Need to mention that the dev dependencies need to be installed to use these

Reply:
I've already added that, its mentioned after the make all command

Reply:
My bad, resolving comment